src/test/crimson/test_socket.cc

```diff
@@ -19,7 +19,7 @@ using ceph::net::Socket;
 using ceph::net::SocketFRef;
 using ceph::net::stop_t;
 
-static seastar::logger logger{"test"};
+static seastar::logger logger{"crimsontest"};
```

Comment:
i was against using subsys for testing. but since we are going to add one anyway. let's just use it. 
```c++
namespace {
  seastar::logger& logger() {
    return ceph::get_logger(ceph_subsys_test);
  }
}
```
and `s/logger/logger()/` in this file.

Reply:
I don't know how the seastar::logger works so I'm not sure what you're saying here, can you push a patch if you don't like what I did?